app/src/main/java/org/fossasia/openevent/app/core/session/list/SessionsFragment.java

```diff
@@ -77,9 +83,21 @@ public void onStart() {
         getPresenter().attach(trackId, this);
         getPresenter().start();
 
+        binding.createSessionFab.setOnClickListener(view -> openCreateSessionFragment());
+
         initialized = true;
     }
 
+    public void openCreateSessionFragment() {
+        //BackStack is popped so that on back press user lands on Dashboard
+        
+        getFragmentManager().popBackStack();
```

Comment:
I think user should end up on Sessions Fragment on back click. It's natural

Reply:
I've made changes so that User lands in `SessionListFragment` on back press but I'm facing an issue here. The user has to press back twice in `SessionListFragment` to get back to `Dashboard` and a toast is displayed which states `"Press back again to exit"` which should not be the case. Can you please suggest any solution ?

Reply:
@iamareebjamal Please have a look at the issue in the above comment.

Reply:
That is a bug in back press handler. 

Flow should be this

dash->tracks->track(sessions) -> create -> track -> tracks -> dashboard -> press back to exit

Reply:
I have made changes such that the flow (dash->tracks->track(sessions) -> create -> track -> tracks(sessions) -> dashboard -> press back to exit ) works as expected but the back press handler is still buggy (sometime two presses are needed to transition back from one fragment to other like, track -> dashboard). Should I create a separate issue for bug press handler ?